tests/Feature/FilterOutgoingLettersTest.php

```diff
@@ -193,4 +193,25 @@ public function user_can_filter_letters_based_on_recipient()
         $this->assertEquals($sentToDUCC[0]->id, $viewLetters[0]->id);
         $this->assertEquals($sentToDUCC[1]->id, $viewLetters[1]->id);
     }
+
+    /** @test */
+    public function user_can_filter_letters_based_on_subject()
+    {
+        $subject1 = factory(OutgoingLetter::class)->create(['subject' => 'Invitation for workshop']);
+        $subject2 = factory(OutgoingLetter::class)->create(['subject' => 'Invitation for event']);
+        $subject3 = factory(OutgoingLetter::class)->create(['subject' => 'Request for extending deadline']);
+
+        $this->be(factory(User::class)->create());
+
+        $viewLetters = $this -> withoutExceptionHandling()
+            ->get('/outgoing-letters?filters[subject][like]=%invitation%')
```

Comment:
Even this is not expected. I would never expect to write `%invitation%` in search box, neither would I write additional scripts for this extra functionality which can be taken care of at back-end.